spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala

```diff
@@ -39,6 +39,7 @@ trait DistributedSparkSuite extends DistributedMahoutSuite with LoggerConfigurat
           .set("spark.kryoserializer.buffer.mb", "15")
           .set("spark.akka.frameSize", "30")
           .set("spark.default.parallelism", "10")
+          .set("spark.driver.allowMultipleContexts", "true")
```

Comment:
@hamnis, could you please explain what is significance of this parameter -- "allowMultipleContexts" in 1.2.0 so that tests won't run?

As far as i remember, current head is using only one context (per suite).so next suite should run another context, but by that time the previous one should already be gone.